src/scripts/mainApp.js

```diff
@@ -75,6 +75,8 @@ function msToTime(duration) {
     return hours + ":" + minutes + ":" + seconds
 }
 
+// FIXME: why are you requiring ipc renderer each time even after you already stored a reference in "var ipc"?
```

Comment:
For some reason it wouldn't receive anything when using the variable, have to dig into this a bit more.